src/box/sql/sqlInt.h

```diff
@@ -3861,44 +3983,7 @@ sql_get_coll_seq(Parse *parser, const char *name, uint32_t *coll_id);
 ssize_t
 sql_index_tuple_size(struct space *space, struct index *idx);
 
-/**
- * Load the content of the _sql_stat1 and sql_stat4 tables. The
- * contents of _sql_stat1 are used to populate the tuple_stat1[]
- * arrays. The contents of sql_stat4 are used to populate the
- * samples[] arrays.
- *
- * @param db Database handler.
- * @retval 0 Success.
- * @retval -1 Error.
- */
-int
-sql_analysis_load(struct sql *db);
-
-/**
- * An instance of the following structure controls how keys
- * are compared by VDBE, see P4_KEYINFO.
- */
-struct sql_key_info {
-	sql *db;
-	/**
-	 * Key definition created from this object,
-	 * see sql_key_info_to_key_def().
-	 */
-	struct key_def *key_def;
-	/** Reference counter. */
-	uint32_t refs;
-	/** Number of parts in the key. */
-	uint32_t part_count;
-	/** Definition of the key parts. */
-	struct key_part_def parts[];
-};
-
-/**
- * Allocate a key_info object sufficient for an index with
- * the given number of key columns.
- */
-struct sql_key_info *
-sql_key_info_new(sql *db, uint32_t part_count);
+struct sql_key_info;
```

Comment:
Lets put struct declarations to the beginning of the file. For instance, to `struct tt_uuid;` which is already there.

Reply:
Fixed.